cpop_bot/text_handler.py

```diff
@@ -36,16 +36,17 @@ async def handle(self, message: types.Message):
             await self.__delete_messages(DELAY_DELETE_IN_SEC_PING,
                                          (pong, message))
 
-    async def __is_provide_service(self, message: types.Message):
+    async def __should_provide_service(self, message: types.Message):
         chat_id = message.chat.id
         if chat_id in WHITELIST_CHAT_ID:
             return True
         chat_type = message.chat.type
         if chat_type == "private":
-            member = await self.bot.get_chat_member(GROUP_CHAT_ID, chat_id)
-            if member.is_chat_member():
-                return True
-            else:
+            user_id = chat_id
```

Comment:
are you sure that **user_id** is **chat_id**?

Reply:
Yes, when the "chat_type" is `private`, the message.from.id is the same value as message.chat.id which is the user's id (the one who was interacting with the bot in private).

it's would be more confusing if it's `(await self.bot.get_chat_member(promotion_chat, chat_id)`

I added the variable to avoid confusing with that.

Reply:
we may could get message.from.id and assign it to the variable "user_id" but I didn't figure out. plus we already have "chat_id" and it's not very confusing that when it's private chat that the chat_id is basically just the user id.

Reply:
oh makes sense, i didn't know that chatId = message.from.id in private. Yeah it's fine to leave it like this then